src/cli.rs

```diff
@@ -10,7 +10,8 @@ use crate::Target;
 
 #[derive(Debug)]
 pub struct Args {
-    pub all: Vec<String>,
+    pub cargo_args: Vec<String>,
+    pub additional_args: Vec<String>,
```

Comment:
```suggestion
    pub args: Vec<String>,
    pub rest_args: Vec<String>,
```

Reply:
im divided on the naming of `cargo_args` and just `args`, however I think `rest_args` is a better name than `additional_args`

Reply:
I've changed it to `rest_args`, however, so far I've kept `cargo_args` since I think the searchability is slightly nicer and it's clearer on the exact functionality of it.